zerver/views/realm.py

```diff
@@ -85,6 +85,15 @@ def check_jitsi_url(value: str) -> str:
         raise JsonableError(_("{var_name} is not an allowed_type").format(var_name=var_name))
 
 
+def check_demo_organization_state(realm: Realm) -> None:
```

Comment:
I wasn't sure about the best place to put this function. It could be added to the realm model. We could also add a `zerver/lib/demo_organizations.py` file or something like that.

Reply:
`zerver/lib/demo_organizations.py` seems like a fine place.

Reply:
Hmm, but I think I'd want a different name. Maybe `check_demo_organization_has_set_email`.